.github/ISSUE_TEMPLATES/bug.yml

```diff
@@ -0,0 +1,23 @@
+name: Bug Report
```

Comment:
I think issue template should also tell go to the extension author and let's remove it from here. The bug itself means that we are not capable to maintain it well.